pkg/identity/certificate_authority.go

```diff
@@ -74,6 +75,32 @@ type FullCAConfig struct {
 	KeyPath  string `help:"path to the private key for this identity" default:"$CONFDIR/ca.key"`
 }
 
+// NewCA creates a new full identity with the given difficulty
+func NewCA(ctx context.Context, opts NewCAOptions) (
+	rv *FullCertificateAuthority, err error) {
```

Comment:
remove line break

Reply:
@jtolds

Reply:
why is this a named return value actually? any objections to replacing it with `_`?

Reply:
my brain is broken. it is not my editor. https://media1.tenor.com/images/74afe6af7eefdda9fbb93e2475b7ad50/tenor.gif?itemid=5082944